### DISCUSSION TITLE
ExplorerEx Beta 0.4

### DISCUSSION THREAD

**Discussion by DearVa** [DearVa/ExplorerEx#43] [Announcements]
# Introducing ExplorerEx Beta 0.4
## The fastest ExplorerEx ever.

![Preview](https://raw.githubusercontent.com/DearVa/ExplorerEx/master/Images/preview.png)

[界面优化；修复视图bug；侧边栏支持一键收起全部](https://github.com/DearVa/ExplorerEx/commit/42da5b8f643d35af987366be2ad8c2a5a5fc1ebb)

[修复了几处侧边栏“此电脑”交互导致软件崩溃的bug](https://github.com/DearVa/ExplorerEx/commit/ca7b5e064189e8b123fc7c913cd63cd7a14c3697)

### [大幅优化文件夹打开速度，特别是诸如System32这种带有大量文件的文件夹。谢谢微软喵！谢谢微软喵！谢谢微软喵！谢谢微软喵！谢谢微软喵！谢谢微软喵！](https://github.com/DearVa/ExplorerEx/commit/98425f33dc3c6196ca425a2380e2b20f72c59bf6)

[修复对话框显示不全的bug；优化设为默认的操作逻辑](https://github.com/DearVa/ExplorerEx/commit/e027ef3589f23ccdd82f440319b51ce0e0ac7eec)

### [通过BHO插件，你可以将ExplorerEx设置为默认文件管理器了！](https://github.com/DearVa/ExplorerEx/commit/c1a1004e507bea5d51bbff713958d5dd87e47cb2)

[修复Windows Explorer重启后托盘图标消失的bug](https://github.com/DearVa/ExplorerEx/commit/79d62598e0f9226dbf967ece15dec33946550048)

[修复粘贴和重命名时会导致文件重复的bug；svg图标预览改用新的开源项目](https://github.com/DearVa/ExplorerEx/commit/c2beb832831114596608b70e330b503f9439e972)

[批量重命名制作中；文件解锁制作中；修复大图标加载的bug；文件重命名提示信息优化；修复视图信息不保存的bug](https://github.com/DearVa/ExplorerEx/commit/3aed9afae402def67a271d504c8b379a8131c373)

### [添加设置新图标；支持背景修改；数据库重构初步完成](https://github.com/DearVa/ExplorerEx/commit/3b3fa0b99bdf9dbc53a33b44cc23312f72637f56)

[优化文件夹监视器；修复如果当前浏览的文件夹被外部删除就会报错的问题；优化打开zip路径的操作逻辑（末尾无需带'\\'）；修复“此电脑”右键菜单仍存在“新建文件夹”和“新建”的问题（点击无反应）；修复“此电脑”右键菜单“显示更多选项”点击直接报错的问题](https://github.com/DearVa/ExplorerEx/commit/65ccc6eae2ed3b858381c364a9b82587345f0f60)

### 感谢 @feast107 喵 [将数据库框架EF Core改为SqlSugar，提升启动速度](https://github.com/DearVa/ExplorerEx/commit/9e91de0e00d1bff2de7d22aff28a920d67b94137)

[升级删除文件的提示框为Fluent样式](https://github.com/DearVa/ExplorerEx/commit/bfe8977c1c69c47b469516b83a0f89ae5d19086f)

[添加设置，默认不显示隐藏的文件夹和受保护的系统文件，如显示会有半透明的视觉效果](https://github.com/DearVa/ExplorerEx/commit/3e88affd11d629c9ed7734d332fc787530f59ea7)

[优化文件夹右键菜单项，添加粘贴按钮；更新部分图标为Fluent样式；修复exe文件不显示缩略图的bug；修复新建文件菜单不显示的bug；修复右键菜单无法新建文件的bug](https://github.com/DearVa/ExplorerEx/commit/641f1c5a182a859c0788d2400b3bb4901d317c48)

[优化图标加载方式，将工作分配到独立线程，减少UI卡顿；优化文件夹加载方式，当文件很多时，会采用惰性加载策略](https://github.com/DearVa/ExplorerEx/commit/92d1713a2d88cb9eabbb67ea7681020ba2e49b05)

[支持直接按下键盘字母或者用输入法输入来快速定位到文件；优化侧边栏样式；修复部分翻译错误](https://github.com/DearVa/ExplorerEx/commit/177e9f38ac9de38a6cedd59b923a7c8d902a123b)

[修复“内容”视图不显示的问题](https://github.com/DearVa/ExplorerEx/commit/8c762f339bd49ea9a330a3f6118e5545c34c9b7b)

### [增加设置页面；亚克力主题回归；窗口右上角控制按钮使用自绘，更加美观；修复右下角视图快捷切换按钮状态不显示、点击无效果的bug](https://github.com/DearVa/ExplorerEx/commit/33076043a6e1974cfde9a601d2a2af04da1b546b)

[修复了右键菜单打开文件夹会在win自带资源管理器打开的bug；修复地址栏不显示下划线的bug；封装ContentDialog](https://github.com/DearVa/ExplorerEx/commit/67c3dd8edd4789e3188492b76fe7f082be417ccd)

[添加解析Path的功能，例如会把%temp%解析成对应的路径](https://github.com/DearVa/ExplorerEx/commit/216272f4db72c38a3dca882e867d27e02366f73e)

<hr /><em>This discussion was created from the release <a href='https://github.com/DearVa/ExplorerEx/releases/tag/beta-0.4'>ExplorerEx Beta 0.4</a>.</em>

**Comment by ghost** [comment]
Wow, best balance of performance and appearance.

**Reply by DearVa** [reply]
Thanks! We have just create our [Discord](https://discord.gg/SB6QFPfnP4) channel and welcome for your coming : )
